src/o365/spfx/commands/project/project-externalize.ts

```diff
@@ -113,28 +114,29 @@ class SpfxProjectExternalizeCommand extends Command {
     Promise
       .all(asyncRulesResults)
       .then((rulesResults) => {
-        this.allFindings.push(...rulesResults.reduce((x, y) => [...x, ...y]));
+        this.allFindings.push(...rulesResults.map(x => x[0]).reduce((x, y) => [...x, ...y]));
```

Comment:
Let's see if this piece will get easier to understand when we move to using an object rather than an array. If not, then let's add some comments for clarity to simplify reading the code and its future maintenance.

Reply:
adding a concrete type for the visitation return should make the reading easier, let me know if you think we should adopt any other changes